### ISSUE TITLE
Replace DB backups/scripts with CI

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#7]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To track database changes in GitHub via Xperience CI.

### Why

Because the database backup/script is a black box from the GitHub perspective. With XML files from the CI feature, we could track changes of a commit without actually checking it out and restoring the full database. Moreover, the checkouts/commits would be significantly smaller and faster.

### Issue expands upon

This issue expands upon another issue(s): #5

### Blockers

This issue is blocked by the following issue(s): #`issue number`
